pkgs/development/python-modules/dmt-core/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, fetchPypi, buildPythonPackage, python3Packages, colorama, colormath
+, cycler, h5py, joblib, matplotlib, more-itertools, packaging, pandas, pint
+, pyarrow, pyqtgraph, pyyaml, requests, scikit-rf, scipy, semver, setuptools
+, reprint, verilogae, pytest }:
+
+buildPythonPackage rec {
+  pname = "dmt-core";
+  version = "2.1.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "DMT_core";
+    hash = "sha256-489E+uNn4NgyCwxsUMEPH/1ZuM+5uNq4zx8F88rkHMU=";
+  };
+
+  propagatedBuildInputs = [
+    colorama
+    colormath
+    cycler
+    h5py
+    joblib
+    matplotlib
+    more-itertools
+    packaging
+    pandas
+    pint
+    pyarrow
+    pyqtgraph
+    pyyaml
+    requests
+    scikit-rf
+    scipy
+    semver
+    setuptools
+    reprint
+    verilogae
+  ];
+
+  nativeBuildInputs = [ pytest ];
+
+  meta = {
+    changelog =
+      "https://gitlab.com/dmt-development/dmt-core/-/blob/Version_${version}/CHANGELOG?ref_type=tags";
+    description =
+      "Python tool to help modeling engineers extract model parameters, run circuit and TCAD simulations and automate infrastructure.";
```

Comment:
```suggestion
      "Python tool to help modeling engineers extract model parameters, run circuit and TCAD simulations and automate infrastructure";
```

Reply:
Also, I'm confused that we are calling it a "tool", but it's evidently not a CLI? We're free to modify the description so long as it explains the project well.

Reply:
I just took the description verbatim but noted.

Reply:
In any case, it shouldn't end with a period: https://nixos.org/manual/nixpkgs/unstable/#var-meta-description

hence my requested change above.